.github/workflows/cd.yml

```diff
@@ -38,19 +51,13 @@ jobs:
       - name: Test
         run: npm run test
 
-      - name: Set release
-        id: semrel
-        uses: go-semantic-release/action@v1
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          allow-initial-development-versions: true
-          force-bump-patch-version: true
-
-      - name: Publish npm package to Artifactory
+      - name: Publish npm package
         run: |
-          export VERSION="${{ steps.semrel.outputs.version }}"
+          export VERSION="${{ needs.release.outputs.version }}"
```

Comment:
Is this "outputs.version"? Didn't you set the name to "outputs.release" above?

Reply:
This takes a value output [here](https://github.com/momentohq/client-sdk-javascript/pull/43/files#diff-ea3ea8c9932adc7ba8161ceda844fedd43b006848ef1140c050cbd7ea0788a18R12-R13).
The same implementation is done in the client_protos' cd workflow [here](https://github.com/momentohq/client_protos/blob/main/.github/workflows/push.yml#L65).